src/scripts/Weather.js

```diff
@@ -0,0 +1,16 @@
+class Weather {
+    constructor(data) {
+        this.geolocation = `${data.location.country}, ${data.location.region}, ${data.location.name}`;
+        this.temperature = `${data.current.temperature}&#8451;`;
+        this.time = `Time: ${data.location.localtime.slice(11, 16)}`;
+        this.feelsLike = `Feels like: ${data.current.feelslike}`;
+        this.today = `Today is ${data.current.weather_descriptions}`;
+        this.wind = `Wind: ${data.current.wind_dir}`;
+        this.windSpeed = `Speed: ${data.current.wind_speed}`;
+        this.plessure = `Plessure: ${data.current.pressure}`;
+        this.cloudcover = `Cloudcover: ${data.current.cloudcover}`;
+        this.icon = `${data.current.weather_icons}`;
+    }
+}
+
+export { Weather };
```

Comment:
```suggestion
const getWeatherData = (data) => ({
        geolocation: `${data.location.country}, ${data.location.region}, ${data.location.name}`,
        temperature: `${data.current.temperature}&#8451;`,
        time: `Time: ${data.location.localtime.slice(11, 16)}`,
        feelsLike: `Feels like: ${data.current.feelslike}`,
        today: `Today is ${data.current.weather_descriptions}`,
        wind: `Wind: ${data.current.wind_dir}`,
        windSpeed: `Speed: ${data.current.wind_speed}`,
        plessure: `Plessure: ${data.current.pressure}`,
        cloudcover: `Cloudcover: ${data.current.cloudcover}`,
        icon: `${data.current.weather_icons}`,
});

export { getWeatherData };
```

Нет смысла создавать клас только для того чтобы вернуть объект